test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3377,27 +3358,29 @@ def test_embedding_bag(self):
         input = torch.randint(10, (7, 5))
         self.run_test(model, (input))
 
-    @disableScriptTest()
+    @disableScriptTest()  # scripting prim::Uninitialized
```

Comment:
nit: prim::Uninitialized is not the only prim op that appears in graph when scripting, maybe add prim:dtype and prim::unchecked_cast in the comment